cdap-ui/app/directives/dag-plus/my-dag-ctrl.js

```diff
@@ -390,6 +393,7 @@ angular.module(PKG.name + '.commons')
 
     function addEndpointForConditionNode(endpointDOMId, endpointStyle, overlayLabel) {
       let endpointDOMEl = document.getElementById(endpointDOMId);
+      endpointStyle.cssClass += ` plugin-${endpointDOMId}`;
```

Comment:
is this only for `no` case? Should we make the cssClass contain the `no`?

Reply:
No the condition node's ID already contains true or false (https://github.com/cdapio/cdap/blob/feature-ui/CDAP-14284-2-pipeline-tests/cdap-ui/app/directives/dag-plus/my-dag-ctrl.js#L338-L339) This should automatically take care of it.